site/app/templates/grading/electronic/NavigationBar.twig

```diff
@@ -89,4 +98,11 @@
         </div>
     </div>
 </div>
-
+<script>
+    //if student/team name(s) are being displayed
+    {# if ({{ not anon_mode and not isBlind and (peer_blind_grading is same as('unblind') or peer_blind_grading is same as('single')) }}) {
+        $('.navigation-box').css({'position':'relative'});
+        $('.navigation-box').css({'float':'left'});
+        $('.navigation-box').css({'margin-left':'20%'});
+    } #}
+</script>
```

Comment:
Pretty sure this commented out JS is here from when I was working on this and should be removed.